tests/PullRequest.py

```diff
@@ -376,3 +376,6 @@ def testAddAndRemoveAssignees(self):
         self.assertListKeyEqual(
             self.pull.assignees, lambda a: a.login, ["jacquev6", "stuglaser"]
         )
+
+    def testUpdateBranch(self):
+        self.pull.update_branch("addaebea821105cf6600441f05ff2b413ab21a36")
```

Comment:
And if you do that, then you can wrap this in self.assertTrue()

Reply:
> And if you do that, then you can wrap this in self.assertTrue()

👍 done